language/move-prover/bytecode-to-boogie/test_mvir/verify-local-ref.mvir

```diff
@@ -0,0 +1,38 @@
+module TestSpecs {
+
+    public mut_b(b: &mut u64){
+        *move(b) = 10;
+        return;
+    }
+
+    public mut_ref()
+      succeeds_if true // force verification of no assertion failures
+    {
+        let b: u64;
+        let b_ref: &mut u64;
+
+        b = 20;
+        b_ref = &mut b;
+        // mutable reference passed into a function call
+        Self.mut_b(copy(b_ref));
+        b = *move(b_ref);
+        assert(move(b) == 10, 42);
+        return;
+    }
+
+    public mut_ref_failure()
```

Comment:
How do you check in our regression testing framework that mut_ref verifies but mut_ref_failure does not?

Reply:
See line 34 -- the `//!` comment contains an expected Boogie error message.

Reply:
Great!  I suppose we are simply grep-ing for that string int the Boogie output without worrying about line numbers since the source line numbers are currently not reflected in the Boogie translation.

Reply:
Yes

Reply:
The current algorithm is sketched in tests/README.md